newsroom/wire/views.py

```diff
@@ -182,7 +185,30 @@ def download(_ids):
     formatter = app.download_formatters[_format]['formatter']
     mimetype = None
     attachment_filename = '%s-newsroom.zip' % utcnow().strftime('%Y%m%d%H%M')
-    if len(items) == 1 or _format == 'monitoring':
+    if formatter.get_mediatype() == 'picture':
+        if len(items) == 1:
+            try:
+                picture = formatter.format_item(items[0], item_type=item_type)
+                return flask.redirect(
+                    url_for('upload.get_upload',
+                            media_id=picture['media'],
+                            filename='baseimage%s' % formatter.get_file_extension()))
+            except ValueError:
+                return flask.abort(404)
+        else:
+            with zipfile.ZipFile(_file, mode='w') as zf:
+                for item in items:
+                    try:
+                        picture = formatter.format_item(item, item_type=item_type)
```

Comment:
seems like this is not used, imo it should be rather:
```py
formatter.get_file_extension(item)
```

Reply:
Passing the file_extension with picture